week-3/extended-project/nowhere/views/index.erb

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Nowhere
+    </title>
+    <!-- Meta tag Keywords -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta name="keywords" content="Hi Codeable!" />
+    <script>
+      addEventListener(
+        'load',
+        function() {
+          setTimeout(hideURLbar, 0);
+        },
+        false
+      );
+
+      function hideURLbar() {
+        window.scrollTo(0, 1);
+      }
+    </script>
+    <!-- //Meta tag Keywords -->
+
+    <!-- Custom-Files -->
+    <link rel="stylesheet" href="css/bootstrap.css" />
+    <!-- Bootstrap-Core-CSS -->
+    <link rel="stylesheet" href="css/style.css" type="text/css" media="all" />
+    <!-- Style-CSS -->
+    <!-- font-awesome-icons -->
+    <link href="css/font-awesome.css" rel="stylesheet" />
+    <!-- //font-awesome-icons -->
+    <!-- /Fonts -->
+    <link
+      href="//fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900"
+      rel="stylesheet"
+    />
+    <link
+      href="//fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://i.imgur.com/3t8FYL1.png"
+      sizes="32x32"
+    />
+
+    <!-- //Fonts -->
+  </head>
+
+  <body>
+    <!-- mian-content -->
+    <div class="main-banner" id="home">
+      <!-- header -->
+      <header class="header">
+        <div class="container-fluid px-lg-5">
+          <!-- nav -->
+          <nav class="py-4">
+            <div id="logo">
+              <h1><a href="/">Nowhere</a></h1>
+            </div>
+
+            <label for="drop" class="toggle">Menu</label>
+            <input type="checkbox" id="drop" />
+            <ul class="menu mt-2">
+              <li class="active"><a href="/">Home</a></li>
+              <li><a href="/about">About</a></li>
+              <li><a href="/team">Team</a></li>
+            </ul>
+          </nav>
+          <!-- //nav -->
+        </div>
+      </header>
+      <!-- //header -->
+      <!--/banner-->
+      <div class="banner-info">
+        <h3>Hi Codeable!</h3>
+        <p class="my-sm-4 my-3">
+          First Bootcamp in Lima-Peru area! <br />
+          Fullstack web developer program. <br />
+          Build awesome stuff using Ruby, React and more.

```

Comment:
In according to Acceptable Criterials," who is on your team and what is Codeable for you" information is not shown in your code